src/components/Camera/FadeInOutView.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import type { Node } from "react";
+import React, { useEffect, useRef } from "react";
+import { Animated, Dimensions } from "react-native";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+
+import viewStyles from "../../styles/camera/fadeInOutView";
+
+const { height } = Dimensions.get( "screen" );
+
+type Props = {
+  savingPhoto: boolean
+}
+
+const fade = value => ( {
+  toValue: value,
+  duration: 100,
+  useNativeDriver: true
+} );
+
+const FadeInOutView = ( { savingPhoto }: Props ): Node => {
+  const fadeAnimation = useRef( new Animated.Value( 0 ) ).current;
+
+  const insets = useSafeAreaInsets( );
+
+  useEffect( ( ) => {
+    if ( savingPhoto ) {
+      Animated.sequence( [
+        Animated.timing( fadeAnimation, fade( 1 ) ),
+        Animated.timing( fadeAnimation, fade( 0 ) )
+      ] ).start( );
+    }
+  }, [savingPhoto, fadeAnimation] );
+
+  const heightPhotoContainerCamera = 134;
```

Comment:
Does this need to be hard-coded? It seems to get repeated [here](https://github.com/inaturalist/iNaturalistReactNative/pull/133/files#diff-ecca3df0533d935137cea989efddaefb2f009aeca3b7f14c8ffd4cf7ca1216ceR85) so maybe it should be extracted, or be some kind of constant in the style.